INSTALL.md

```diff
@@ -148,7 +148,7 @@ tarball and use it to install Singularity:
 $ cd $GOPATH/src/github.com/sylabs/singularity && \
   ./mconfig && \
   make -C builddir rpm && \
-  sudo rpm -ivh ~/rpmbuild/RPMS/x86_64/singularity-3.0.3-687.gf3da9de.el7.x86_64.rpm # or whatever version you built
+  sudo rpm -ivh ~/rpmbuild/RPMS/x86_64/singularity-3.5.3-687.gf3da9de.el7.x86_64.rpm # or whatever version you built
```

Comment:
The "version" changed ... but that doesn't change the rest of the RPM name ... so copy/paste of this will fail unless you just happen to build 687 commits after the 353 tag, and you just happen to get a GIT hash of f3da9de ... 

Maybe change that to be like:
```
sudo rpm -ivh ~/rpmbuild/RPMS/x86_64/singularity-3.5.3*.x86_64.rpm
```

Reply:
Yeah - I thought that was the purpose of the `# or whatever version you built` as that commit hash is wrong except for one point in time.

Reply:
Well ... You changed other version numbers where there was a `# or whatever version` comment as well. ;)

Reply:
yeah - and I changed the version here... just not the git commit suffix bit ;-)

I'll put the `*` in in a bit :-)